src/ImageRunModal.jsx

```diff
@@ -633,6 +588,79 @@ export class ImageRunModal extends React.Component {
         return owner === systemOwner;
     };
 
+    isFormValid = (validationFailed) => {
+        return Object.keys(validationFailed).length == 0;
+    };
+
+    async validateContainerName(containerName) {
+        try {
+            await client.containerExists(this.isSystem(), containerName);
+        } catch (error) {
+            return;
+        }
+        return _("Name already in use");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20231009-110933-d1e7ff0e-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)